packages/owl-bot/test/docker-api.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {describe, it} from 'mocha';
+import * as docker from '../src/docker-api';
+import * as assert from 'assert';
+
+describe.skip('fetchConfig', () => {
```

Comment:
Checking in a test with a `.skip` don't feel quite right.  Do we really want a test that performs a network operation?  Could we use nock to return a mocked response here?

Reply:
Yes, I want a test that performs a network operation.  I'm implementing one half of a protocol.

If I were to drop the integration test and write a test with nock, then I'm very unlikely to catch a bug.  The test would only confirm that my (mis?)understanding of the client agrees with my (mis?)understanding of the server.  It would not confirm that either agrees with the defined protocol.

So yes, I want check-in this integration test, so that if I change the code later, I can remove the `.skip` and at least confirm that it works on my machine.

I am open to adding another test, a unit test that uses nock and is not skipped.  Would you like to see that?

Reply:
Can we create a separate integration-test folder and add a separate `npm run` command to run the integration tests?

Reply:
Done.

Ready for another look.